src/main/webapp/script.js

```diff
@@ -55,16 +55,11 @@ function createListElementUser(drink) {
   ratingElement.className = 'drink';
 
   const drinkElement = document.createElement('span');
-  var store = idToStore(drink.store)
-  drinkElement.innerText = store +", "+drink.drink + ", " + drink.rating + "/5, " + drink.content;
+  drinkElement.innerText = drink.storeInfo +", "+drink.drink + ", " + drink.rating + "/5, " + drink.content;
```

Comment:
Please attach a couple screenshots to show how the comment is rendered on the TYD Coffee UI.

Reply:
https://screenshot.googleplex.com/8h0pGxskcr5